typescript/nft-multiple-currencies/pages/api/generate-mint-sig.ts

```diff
@@ -0,0 +1,46 @@
+import { ThirdwebSDK } from "@thirdweb-dev/sdk";
+import { ethers } from "ethers";
+import type { NextApiRequest, NextApiResponse } from "next";
+import { nft } from "../../nft";
+
+const generateMintSignature = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+) => {
+  const { tokenId, address, tokenName } = JSON.parse(req.body);
+
+  const sdk = new ThirdwebSDK(
+    new ethers.Wallet(
+      process.env.PRIVATE_KEY as string,
+      ethers.getDefaultProvider(process.env.ALCHEMY_API_URL)
+    )
+  );
+
+  const edition = sdk.getEdition("0x4ebCf39BCeEA3C9790605BF292B4a2b8811C93eD");
+
+  const token = nft.tokens.find((t) => t.name === tokenName)
```

Comment:
```suggestion
  const token = nft.tokens.find((t) => t.name === tokenName)
```

`nft.tokens.find((t) => t.name === tokenName)` probably better in a separate variable instead of repeating it twice (not super important)